.gitattributes

```diff
@@ -0,0 +1,8 @@
+/tests export-ignore
```

Comment:
well, it actually make sense. We want to exclude the tests folder at the root of the repo, but not folders named tests which could appear elsewhere (for code using a ``Tests`` namespace to provide test helpers for libraries for instance)